data_preserve.rb

```diff
@@ -0,0 +1,80 @@
+require 'json'
+
+RENTALS_PATH = 'rentals.json'.freeze
+BOOKS_PATH = 'books.json'.freeze
+PEOPLE_PATH = 'people.json'.freeze
+
+class DataPreserve
+  def self.save(books, people, rentals)
+    save_books(books)
+    save_people(people)
+    save_rentals(rentals)
+  end
+
+  def self.reload
+    reload_books
+    reload_people
+    reload_rentals
+    [@books, @people, @rentals]
+  end
+
```

Comment:
- [ ] After exiting the application, the data was saved :+1: . However, when attempting to load the saved data by running the application, I encountered the following error as shown below  👇 so please make sure that you fix this error and ensure that the application still works as expected :pray: 

  ![image](https://github.com/andrianarivo/oop_school_library/assets/92303891/26368f11-fbeb-49b9-a3d7-c42f2c2c8073)

Reply:
Hi! @Owusu-Desmond , thank you for your review! ^^ It's because of rubocop. It changed the `JSON.load` method to `JSON.parse`. It was really late last night and we forgot to test one last time. It's fixed now!